cmp/050f753f-1323-43ae-ad72-3f23d7567393/component.lp

```diff
@@ -0,0 +1,41 @@
+(librepcb_component 050f753f-1323-43ae-ad72-3f23d7567393
+ (name "DAC I²C Single-Ended")
+ (description "A generic single-ended single-channel I²C digital-to-analog converter.")
+ (keywords "dac,i2c,single-ended,single-channel")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2018-12-02T14:10:04Z)
+ (deprecated false)
+ (category f87a2a14-4dd2-49f0-b93d-0c78ac50cf53)
+ (schematic_only false)
+ (default_value "{{ PARTNUMBER or DEVICE or COMPONENT }}")
```

Comment:
It's not a (manufacturer-)specific component, thus it doesn't make sense to see its name in the schematics. Also remove spaces for consistency.
```suggestion
 (default_value "{{PARTNUMBER or DEVICE}}")
```

Reply:
That was actually copied from [the docs](https://developers.librepcb.org/de/df4/doc_attributes_system.html), so you should probably remove the inner spaces there too :slightly_smiling_face:

Reply:
Ah true, even within the docs it's not consistent :(